<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventLauncher</class>
 <widget class="QDialog" name="EventLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Launcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select the event to log</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent1">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent2">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent3">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent4">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent5">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent6">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 6</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent7">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 7</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbEvent8">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Event 8</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pbCancel">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
